tests/test_cell_conditional_formatting.py

```diff
@@ -0,0 +1,68 @@
+import dash_ag_grid as dag
+from dash import Dash, html, dcc
+from . import utils
+import time
+
+def test_fi001_cell_conditional_formatting(dash_duo):
```

Comment:
```suggestion
def test_cf001_cell_conditional_formatting(dash_duo):
```
The idea here is to make that first code after `test_` unique within the repo, so if you want to run just this one test locally you can do `pytest -k <test-case-id>`.

For this repo I'd like to use the convention of two letters for the file, followed by a three-digit serial number within the file (other, bigger repos use four letters + three digits - two letters for the folder or component, two for the file, three-digit serial number) So this file would be `cf` for conditional formatting.